src/main/java/envoy/client/ui/controller/ContactSearchScene.java

```diff
@@ -61,7 +63,12 @@ private void initialize() {
 		contactList.setCellFactory(ContactListCellFactory::new);
 		searchBar.setClearButtonListener(e -> { searchBar.getTextField().clear(); contactList.getItems().clear(); });
 		eventBus.register(ContactSearchResult.class,
-				response -> Platform.runLater(() -> { contactList.getItems().clear(); contactList.getItems().addAll(response.get()); }));
+				response -> Platform.runLater(() -> {
+					List<Chat> chats = new ArrayList<Chat>();
+					response.get().stream().forEach(r -> chats.add(new Chat(r)));
+					contactList.getItems().clear();
+					contactList.getItems().addAll(chats);
```

Comment:
That can be rewritten as:
```suggestion
					contactList.getItems().clear();
					contactList.setItems(response.get().stream().map(Chat::new).collect(Collectors.toList()));
```
, can't it?

Reply:
No this lambda did not work, as the `setItems()` function takes an `ObservableList`. I changed it to`contactList.getItems().addAll(response.get().stream().map(Chat::new).collect(Collectors.toList()));` and now it works, but the basic concept of your suggestion was good👍

Reply:
To be fair, I forgot something:
```suggestion
					contactList.getItems().clear();
					contactList.setItems(FXCollections.observableList(response.get().stream().map(Chat::new).collect(Collectors.toList())));
```
It doesn't matter however as your solution does basically the same.